idmapping-rest/src/main/java/org/uniprot/api/idmapping/controller/request/IdMappingJobRequestToArrayConverter.java

```diff
@@ -0,0 +1,28 @@
+package org.uniprot.api.idmapping.controller.request;
+
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import org.uniprot.api.rest.request.idmapping.IdMappingJobRequest;
+import org.uniprot.core.util.Utils;
+
+public class IdMappingJobRequestToArrayConverter implements Function<IdMappingJobRequest, char[]> {
+    @Override
+    public char[] apply(IdMappingJobRequest request) {
+        StringBuilder builder = new StringBuilder();
+        builder.append(request.getFrom().strip().toLowerCase());
+        builder.append(request.getTo().strip().toLowerCase());
+        builder.append(
+                List.of(request.getIds().strip().split(",")).stream()
```

Comment:
Minor Comment: 
You CAN replace List.of(request.getIds().strip().split(",")).stream() with Stream.of(request.getIds().strip().split(","))

(Can be seen from IDE suggestions as well)